guides/common/modules/ref_capsule-ports-and-firewalls-requirements.adoc

```diff
@@ -80,19 +80,22 @@ endif::[]
 |====
 
 
-
 .Ports for {SmartProxy} to Client Communication
 [cols="24%,20%,18%,38%a",options="header"]
 |====
 | Port | Protocol | Service | Required For
-| 7 | TCP and UDP | ICMP | DHCP {SmartProxy} to Client network, ICMP ECHO to verify IP address is free (Optional)
+| - | ICMP | ping | DHCP {SmartProxy} to Client network, to verify active IP address (Optional)
+| 7 | TCP | echo | DHCP {SmartProxy} to Client network, to verify active IP address (Optional)
 | 68 | UDP | DHCP | {SmartProxy} to Client broadcasts, DHCP broadcasts for Client provisioning from a {SmartProxy} (Optional)
 | 8443 | TCP |HTTP | {SmartProxy} to Client "reboot" command to a discovered host during provisioning (Optional)
 |====
 
 Any managed host that is directly connected to {ProjectServer} is a client in this context because it is a client of the integrated {SmartProxy}.
 This includes the base operating system on which a {SmartProxyServer} is running.
 
+A DHCP {SmartProxy} performs ICMP ping or TCP echo connection attempts to hosts in subnets with DHCP IPAM set to find out if an IP address considered for use is active.
+This behavior can be turned off via {foreman-installer}.
```

Comment:
Thanks a lot for your input, Lzap. Can you please provide the exact command that manipulates this behavior?

Reply:
Is this like "choose either a ICMP ping or a TCP echo"?

Reply:
In Foreman 1.24 it was added as an option to disable it (prior to that it was always on) and since Foreman 2.4 this is exposed in the installer as `--foreman-proxy-dhcp-ping-free-ip`. (See https://projects.theforeman.org/issues/31415.) There is no way to turn off ICMP or TCP invidually - it's either both or none. That means this single line should be cherry picked to 2.4 docs, but not older. The rest is all relevant to older versions.

Reply:
And what happens if you disable it?

Reply:
If the backend (whether it's ISC DHCP) or Infoblox says an IP is free, the Smart Proxy believes that. In a properly maintained environment where everything is tracked, the extra free check is redundant. However, if (for example) a server has an additional static IP that's not registered anywhere but does respond on the network then you would have a conflict. In brown field deployments this is common which is why the additional check was added way back when.